### DISCUSSION TITLE
How do I get GitHub PRO?

### DISCUSSION THREAD

**Discussion by mixed-farming** [community/community#69005] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it possible to get GitHub PRO for free? What are the perks of getting GitHub PRO?

**Comment by SangeethaPrabhu2021** [comment]
One way to get GitHub Pro for free if you are a college student, you can get it through the Student Developer Pack. Please click on the link below to Sign up for it...!
https://education.github.com/pack

Unlimited Private Repositories: GitHub Pro allows you to create an unlimited number of private repositories. Free GitHub accounts are limited to a certain number of private repositories.

Collaboration Tools: GitHub Pro includes collaboration features such as team and user mentions, code review tools, and issue tracking, which can be helpful for project management and collaboration with others.

Advanced Code Review Tools: GitHub Pro offers more advanced code review tools and features, such as code review assignments, pull request templates, and multiple reviewers.

Access Control: You have more control over access to your repositories, allowing you to manage permissions and restrict access to certain team members or collaborators.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
